tools/analyzer_plugin/lib/src/async_plugin_apis/diagnostic.dart

```diff
@@ -147,59 +150,71 @@ class _DiagnosticGenerator {
   /// by the given [unitResult]. If any of the contributors throws an exception,
   /// also create a non-fatal 'plugin.error' notification.
   Future<_GeneratorResult<List<AnalysisError>>> generateErrors(ResolvedUnitResult unitResult) async {
-    return _generateErrors(unitResult, DiagnosticCollectorImpl(shouldComputeFixes: false));
+    return _generateErrors(unitResult, DiagnosticCollectorImpl(shouldComputeFixes: false),
+        catchInconsistentAnalysisException: true);
   }
 
   /// Creates an 'edit.getFixes' response for the location in the file specified
   /// by the given [request]. If any of the contributors throws an exception,
   /// also create a non-fatal 'plugin.error' notification.
   Future<_GeneratorResult<EditGetFixesResult>> generateFixesResponse(DartFixesRequest request) async {
```

Comment:
I'd recommend using [whitespace-agnostic diff](https://github.com/Workiva/over_react/pull/871/files?w=1#diff-7a8e28d75e77fe3c8eda22871a60c2536a21744c2746b7ec68644913d62eeff1R160) for these diffs